omero/users/clients-overview.txt

```diff
@@ -56,7 +56,12 @@ views.
     :alt: OMERO.web image viewer
     
     OMERO.web image viewer
-    
+
+OMERO.web now features almost all of the functionality of OMERO.insight
```

Comment:
may want to make it clear this doesn't include import lest readers become prematurely excited